### ISSUE TITLE
How to configure the path mapping for a folder with the site, and not for the folder with Devilbox?

### ISSUE THREAD

**Issue by evgeny-2019** [cytopia/devilbox#820]
#### ISSUE TYPE
 - Question

#### SUMMARY
In the documentation on this page:
https://devilbox.readthedocs.io/en/lateest/intermediate/configure-php-xdebug/windows/phpstorm.html#configure-php-xdebug-win-phpstorm
The devilbox folder is open as a project. And Path Mapping is configured for it.

I would like to open a specific site folder as a project, which is located in data/www/. So that the IDE not indexed folders with other sites.

How can I configure Path Mapping for this?

#### Goal
Configure Path Mapping for a folder with the site, and not for a folder with Devilbox.

**Comment by cytopia** [comment]
@evgeny-2019 can you rephrase this. I didn't quite understand what you mean.

**Comment by cytopia** [comment]
@evgeny-2019 any feedback on this?

**Comment by cytopia** [comment]
@evgeny-2019 Closing this issue as it was not clear what you wanted to achieve and it has been open for a long time without clarification. Feel free to add more info and I could re-open
